nncf/quantization/advanced_parameters.py

```diff
@@ -152,7 +152,7 @@ class AdvancedQuantizationParameters:
     overflow_fix: OverflowFix = OverflowFix.FIRST_LAYER
     quantize_outputs: bool = False
     inplace_statistics: bool = True
-    disable_channel_alignment: bool = True
+    disable_channel_alignment: bool = False
```

Comment:
Please, discuss influence of this change on `nncf.quantize_with_accuracy_control(...)` with @andrey-churkin.

Reply:
These changes do not affect `nncf.quantize_with_accuracy_control(...)` when `tune_hyperparams` is set to `False`. However, when `tune_hyperparams` is set to `True`, we cannot apply hyperparameter tuning due to this check.

https://github.com/openvinotoolkit/nncf/blob/2fe6ee3c94e5f7875926a8722e2b98f2e4e1ad28/nncf/quantization/algorithms/post_training/algorithm.py#L166-L170

Reply:
@andrey-churkin, what should I do to make `tune_hyperparams` works again?